crates/chia-sdk-bindings/src/mips/spend.rs

```diff
@@ -268,21 +281,30 @@ impl MipsSpend {
         let nonce = config.nonce.try_into().unwrap();
         let restrictions = convert_restrictions(config.restrictions);
 
-        let member = SingletonMember::new(launcher_id);
-
-        let member_hash = mips_puzzle_hash(
-            nonce,
-            restrictions.clone(),
-            member.curry_tree_hash(),
-            config.top_level,
-        );
+        let (member_hash, member_puzzle) = if fast_forward {
```

Comment:
for all of these `if fast_forward ...` ternaries, I'd prefer it apply just to the instantiation of `member` rather than the whole block, since member is all that is changing so we're duplicating the other pieces.

n.b.d., just my 2c